praw/models/reddit/subreddit.py

```diff
@@ -1393,6 +1405,19 @@ def update(
             "text_color": text_color,
             "text_editable": text_editable,
         }
+        if fetch:
+            _existing_data = [
+                template
+                for template in iter(self)
+                if template["id"] == template_id
+            ]
+            if len(_existing_data) == 0:
+                raise InvalidFlairTemplateID(template_id)
```

Comment:
As noted previously, I'd like this to be a `DeprecationWarning` for now. The code should still do something meaningful until this is officially removed.

Reply:
I am aware, but because we are pushing to v7.0.0, I decided to implement the major breaking API changes back.